Tutorial/3_EspressoMachine/PSrc/CoffeeMakerControlPanel.p

```diff
@@ -172,16 +172,14 @@ machine CoffeeMakerControlPanel
 
   state CoffeeMakerDoorOpened {
     on eCloseGroundsDoor do {
-      if(cofferMakerState == NotWarmedUp)
-        goto WarmUpCoffeeMaker;
-      else
-        goto CoffeeMakerReady;
+      assert cofferMakerState != NotWarmedUp;
```

Comment:
@uztadh

Can we add an assertion `assert cofferMakerState == Ready;` here?

Based on the entry function of state `CoffeeMakerReady` (line 84 of this file) and the only state transition to state `CoffeeMakerDoorOpened` on line 88, `cofferMakerState` should be in `Ready` state.

Reply:
on it

Reply:
Okay, I've added the assertion